Library/Homebrew/extend/os/linux/dev-cmd/update-test.rb

```diff
@@ -6,6 +6,8 @@ module Homebrew
 
   module_function
 
+  alias generic_git_tags git_tags
+
   def git_tags
     tags = generic_git_tags
```

Comment:
Not sure why it's saying `generic_git_tags` is undefined here.

Reply:
@apainintheneck I think you need to make the alias in `Library/Homebrew/dev-cmd/update-test.rb` instead, that's what we've done previously here.

Reply:
That's actually what I tried first and I moved after seeing the same error. I feel like I'm missing something obvious here.